### DISCUSSION TITLE
How to disable user interaction when tour is temporarily paused?

### DISCUSSION THREAD

**Discussion by RHOOPH** [gilbarbara/react-joyride#906] [Q&A] [answered]
As in the [controlled demo](https://react-joyride.com/controlled) , we pause the tour when clicking on the menu, then after a delay, we restart it again, How can we prevent user from taking any action during the delay, 

Or in similar situations where the delay is noticeably long,  Is there a built-in way to do it? or go for custom implementation?

**Comment by gilbarbara** [comment, marked as answer]
Hey @RHOOPH 

No, there isn't a built-in way to do it.
You could render an absolutely positioned div covering the whole page with a high z-index and pointer-events: none.
